common/spinner.py

```diff
@@ -25,7 +25,7 @@ def update(self, spinner_text: str):
         pass
 
   def update_progress(self, cur: int, total: int):
-    self.update(str(int(100 * cur / total)))
+    self.update(str(round(100 * cur / total)))
```

Comment:
why change this? strongly doubt the difference is noticeable

Reply:
Just so it's more precise, int doesn't round. Something that's 99.9 won't make the spinner display 100%. It's a small change but more correct I think